tests/testthat/test-5dominantTaxa.R

```diff
@@ -0,0 +1,57 @@
+context("dominantTaxa")
+
+test_that("dominantTaxa", {
+
+    # TSE object
+    data(GlobalPatterns)
+    tse <- GlobalPatterns
+
+    # Phyloseq object
+    data("GlobalPatterns", package = "phyloseq")
+    phy <- GlobalPatterns
+
+    # Counts table should not be changed
+    expect_equal(assays(mia::dominantTaxa(tse))$counts, assays(tse)$counts)
+
+    # Should get same result
+    expect_equal(colData(mia::dominantTaxa(tse, name="dominant"))$dominant, microbiome::dominant(phy))
```

Comment:
You shouldn't use the microbiome package in the test, because then you need to put in the DESCRIPTION, which then pulls in `phyloseq`. It does not make sense. I think an approach with fixed values is OK or compute it fixed with 
`rownames(mat)[apply(mat, 2, which.max)]`

Reply:
Used fixed values that I got with microbiome and microbiomeutilities packages